### PR TITLE
docs: minor improvements

### PR DESCRIPTION
**Pull request by saragerion**
## Description of your changes

Changes introduced:
- Clarification that people can use the library in both TypeScript and JavaScript code bases
- References to Serverless TypeScript Demo
- One-liner command to install all three utilities at once 
- Contribute CTA
- Driving our roadmap CTA
- Consistency with messaging of blog post (utilities key features etc)
- Changed font size and remove ToC on the right side to improve readability in smaller screens

### How to verify this change

Docs:  npm run docs-runLocalDocker

### Related issues, RFCs

N/A

### PR status

***Is this ready for review?:*** YES  
***Is it a breaking change?:*** NO

## Checklist

- [x] [My changes meet the tenets criteria](https://awslabs.github.io/aws-lambda-powertools-python/#tenets)
- [x] I have performed a *self-review* of my own code
- [ ] I have *commented* my code where necessary, particularly in areas that should be flagged with a TODO, or hard-to-understand areas
- [x] I have made corresponding changes to the *documentation*
- [ ] I have made corresponding changes to the *examples*
- [x] My changes generate *no new warnings*
- [x] The *code coverage* hasn't decreased
- [ ] I have *added tests* that prove my change is effective and works
- [x] New and existing *unit tests pass* locally and in Github Actions
- [x] Any *dependent changes have been merged and published* in downstream module
- [x] The PR title follows the [conventional commit semantics](https://github.com/awslabs/aws-lambda-powertools-typescript/blob/main/.github/semantic.yml#L2)

---

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -67,7 +67,6 @@ You can instrument your code with Powertools in three different ways:
 | **POWERTOOLS_TRACER_CAPTURE_HTTPS_REQUESTS** | Captures HTTP(s) requests as segments.                                                 | [Tracer](./core/tracer)   | `true`                |
 | **POWERTOOLS_LOGGER_LOG_EVENT**              | Logs incoming event                                                                    | [Logger](./core/logger)   | `false`               |
 | **POWERTOOLS_LOGGER_SAMPLE_RATE**            | Debug log sampling                                                                     | [Logger](./core/logger)   | `0`                   |
-| **POWERTOOLS_LOGGER_LOG_EVENT**              | Logs incoming events                                                                   | [Logger](./core/logger)   | `false`               |
```
Comment (saragerion) [inline docs/index.md:53]: Redundant

**Review by dreamorosi — APPROVED** [review]
LGTM!

**Review by ijemmy — COMMENTED** [review]
Not part of this PR. But should we consider having a dedicated README for each utility?

Copying the same README to every packages makes me feel uncomfortable. I'd prefer to simply put a link the official doc for the package so that we have only a single source of documents.  The README at this level should be specific to contribution/development.

**Review by ijemmy — APPROVED** [review]

**Comment by saragerion** [comment]
@ijemmy completely agree and thanks for bringing this up.

**Pull request merged**